Mage/src/main/java/mage/abilities/keyword/FreerunningAbility.java

```diff
@@ -34,6 +35,13 @@ public FreerunningAbility(String manaString) {
         this.addHint(FreerunningCondition.getHint());
     }
 
+    public FreerunningAbility(Cost cost) {
+        super(FREERUNNING_KEYWORD, FREERUNNING_REMINDER, cost);
```

Comment:
you should change the other constructor to call this new constructor so there's only one constructor that calls the super